From: Peter Lemenkov <lemenkov@gmail.com>
Date: Fri, 4 Mar 2016 17:54:14 +0300
Subject: [PATCH] Proper version in app-file

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>

diff --git a/src/riak_pb.app.src b/src/riak_pb.app.src
index d1f07af..ccc6476 100644
--- a/src/riak_pb.app.src
+++ b/src/riak_pb.app.src
@@ -1,7 +1,7 @@
 {application, riak_pb,
  [
   {description, "Riak Protocol Buffers Messages"},
-  {vsn, git},
+  {vsn, "2.1.2.0"},
   {registered, []},
   {applications, [
                   kernel,
